installer-scripts/swift-inspect.wixproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="WixToolset.Sdk/4.0.0">
```

Comment:
I think that we should consider upstreaming this into apple/swift-installer-scripts as this is something that will be in the official toolchain releases.  If this is not in the official repos we cannot have this in the toolchain releases.

Reply:
Yep. Is it okay to merge this here first and then move it to apple/swift-installer-scripts? We cannot atomically commit across two repos, right? Or do we need to merge it into apple/swift-installer-scripts first and then merge this?

Reply:
I think that since you have tested the build, lets get the changes merged into apple/swift-installer-scripts first and then use them here.

Reply:
Sent https://github.com/apple/swift-installer-scripts/pull/204 for the wix files.